src/api/views/payment/checkout_webhook_view.py

```diff
@@ -0,0 +1,103 @@
+                               status=status.HTTP_400_BAD_REQUEST)
+
+    licence_plate: LicencePlate = LicencePlate.objects.get(user=User.objects.get(pk=user_id),
+                                                           licence_plate=licence_plate)
+
+    licence_plate.updated_at = timezone.now()
+
+    try:
+        licence_plate.save()
+        return BackendResponse(['Completed order'], status=status.HTTP_200_OK)
+    except:
+        return BackendResponse(['Failed to save new updated_at time.'], status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+class CheckoutWebhookView(APIView):
+    """
+    A view to listen for checkout updates from the stripe servers.
+    """
+
+    permission_classes = [AllowAny]
+
+    def post(self, request: Request, format=None) -> BackendResponse:
+
+        if 'STRIPE_SIGNATURE' not in request.headers:
+            return BackendResponse(['This endpoint is only accessible by Stripe.'], status=status.HTTP_403_FORBIDDEN)
+
+        sig_header = request.headers['STRIPE_SIGNATURE']
+        payload = request.body
+
+        try:
+            event = stripe.Webhook.construct_event(
+                payload, sig_header, endpoint_secret
+            )
+        except ValueError as e:
+            # Invalid payload
+            return BackendResponse([str(e)], status=status.HTTP_400_BAD_REQUEST)
+        except stripe.error.SignatureVerificationError as e:
+            # Invalid signature
+            return BackendResponse([str(e)], status=status.HTTP_400_BAD_REQUEST)
+
+        # Handle the checkout.session.completed event
+        if event['type'] == 'checkout.session.completed':
+            session = event['data']['object']
+
+            # Fulfill the purchase...
+            complete_order(session)
+
+        # Handle the checkout.session.completed event
+        if event['type'] == 'checkout.session.completed':
+            session = event['data']['object']
+
+            # Save an order in your database, marked as 'awaiting payment'
+            # create_order(session)
+
+            # Check if the order is already paid (for example, from a card payment)
+            #
+            # A delayed notification payment will have an `unpaid` status, as
+            # you're still waiting for funds to be transferred from the customer's
+            # account.
+            if session.payment_status == "paid":
+                # Fulfill the purchase
+                complete_order(session)
+
+        elif event['type'] == 'checkout.session.async_payment_succeeded':
+            session = event['data']['object']
+
+            # Fulfill the purchase
+            complete_order(session)
+
+        elif event['type'] == 'checkout.session.async_payment_failed':
+            session = event['data']['object']
+
+        # Passed signature verification
+        return BackendResponse(["Order is fulfilled."], status=status.HTTP_200_OK, )
```

Comment:
There is no need to wrap this in a list. Errors have to be `list[str]` by convention and normal operation returns `data{...}`.